.gitignore

```diff
@@ -1,2 +1,4 @@
 *~
 _build/
+_static/
+_templates/
```

Comment:
What is the reason for ignoring them?  I think right now they are empty (except of a `.keep` file for git) but they are generally part of the documentation source files.